### PR TITLE
the docker setup

### PR DESCRIPTION
**Pull request by Thekid2002**

### CODE REVIEW

**Comment by DeeKahy** [comment]
Please provide an explanation to this pull request, and follow the standard for pr naming.

File: server/moodledata/temp/theme/boost/all.css
Comment (DeeKahy) [inline server/moodledata/temp/theme/boost/all.css:1]: This file shouldn't be committed.

`git update-index --assume-unchanged (the file location)` should fix the problem if git is still tracking it.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      #Remove this line for production testing
+      - ./server/moodle/mod/livequiz:/var/www/html/mod/livequiz
+
+  db:
+    image: postgres:16
+    container_name: moodle-postgres-db
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - ./initdb.sql:/docker-entrypoint-initdb.d/initdb.sql
+    environment:
```
Comment (DeeKahy) [inline docker-compose.yml:24]: If possible, I'd prefer if we stick with the same database provided by Ida (mariadb or mysql).

Comment (Thekid2002) [reply docker-compose.yml:24]: If possible, I'd prefer we did not. Men vi kan lige snakke om det, ellers bliver det på dagsordenen til næste PO møde 😄  Der er mange fordele, flere stod også på dagsordenen for dagens PO møde 😸

Comment (DeeKahy) [reply docker-compose.yml:24]: I know Postgres is the better database system. But Ida mentioned today during the PO meeting today that everything needs to get merged by her, so it should follow the general systems she has outlined herself.

Comment (Thekid2002) [reply docker-compose.yml:24]: Jeg vil helst tage diskussionen ansigt til ansigt. 😄  For jeg tror måske ida tror at hvis vi bruger postgres til udvikling kan hun ikke længere bruge HeidiSql, men dette er en forkert antagelse. Jeg er åben for yderligere diskussion, men dette skal helst forgå gennem min PO Christoffer, eller ansigt til ansigt 😄

Comment (DeeKahy) [reply docker-compose.yml:24]: Understood.

This pr is stalled until the next PO meeting with Ida.

Comment (Thekid2002) [reply docker-compose.yml:24]: Eller indtil yderligere diskussion 👯

**Comment by DeeKahy** [comment]
Love the setup and it would help basically everyone.

Waiting for a meeting about switching database from MariaDB to Postgres with Ida to make sure everybody is on board.

**Comment by DeeKahy** [comment]
Okay ive taken a closer look at the pull request. At first, I assumed it was just a docker image to properly launch the repository, but it looks like it is the same as the `Præsentation af et containerized setup` that is described below. Is that correct? @Thekid2002 

```
Præsentation af et containerized setup
Repository’et kan godt hurtigt komme til at fylde meget, blive rodet og i det hele taget være svært at finde hoved og hale i. Derfor foreslår vi at der bliver lavet et nyt repository udelukkende til pluginet og altså ikke også Moodle koden. Dette er muligt, da den nyligt containerized version af Moodle kan overtage pladsen fra Moodle. Denne løsning medbringer flere fordele, blandt andet kræver det mindre installation, virker på flere system arkitekturer og tager kortere tid for alle at sætte op.
```

**Comment by DeeKahy** [comment]
I'm closing it because of inactivity and the failure to comply with the naming conventions I requested originally.

**Pull request closed without merging**